vault/expiration.go

```diff
@@ -1029,7 +1052,8 @@ func (m *ExpirationManager) RevokePrefix(ctx context.Context, prefix string, syn
 // RevokeByToken is used to revoke all the secrets issued with a given token.
 // This is done by using the secondary index. It also removes the lease entry
 // for the token itself. As a result it should *ONLY* ever be called from the
-// token store's revokeSalted function.
+// token store's revokeInternal function.
+// (NB: it's called by token tidy as well.)
```

Comment:
what does NB stand for? Pratt just asked me this yesterday too

Reply:
https://en.wikipedia.org/wiki/Nota_bene

Reply:
thanks! i'd never seen that before